README.md

```diff
@@ -1,30 +1,109 @@
-FuseSoC Dockerfiles
-====
-
-This repository contains several packages for various kinds of automation with [FuseSoC](https://github.com/olofk/fusesoc).
-
-**Disclaimer:** All available images are available for evaluation and testing **only**.
-Location of this images may change along with migration to a GitHub organization.
-
-## Available images
-
-* [onenashev/fusesoc-icarus](https://hub.docker.com/r/onenashev/fusesoc-icarus/) - 
-Bundles FuseSoC and [Icarus Verilog](http://iverilog.icarus.com/)
- * Image can be used for basic simulations of projects listed in [orpsoc-cores](https://github.com/openrisc/orpsoc-cores)
-* [onenashev/fusesoc-icarus-quartus](https://hub.docker.com/r/onenashev/fusesoc-icarus-quartus/) - 
-Bundles FuseSoC, [Icarus Verilog](http://iverilog.icarus.com/) and Quartus II Web Edition 15.1.2
- * With this image it is possible to build systems based on Altera FPGAs ([list of available systems](https://github.com/openrisc/orpsoc-cores/tree/master/systems))
- 
- 
- ## Usage examples
- 
- Both available images wrap FuseSoC, hence you can just invoke commands similarly to common CLIs.
- 
- ```
- docker run onenashev/fusesoc-icarus fusesoc list-cores
- ```
- 
- ```
- docker run onenashev/fusesoc-icarus fusesoc sim wb_sdram_ctrl
- ```
- 
\ No newline at end of file
+# LibreCores Tools Docker Images
+
+This repository contains the information about tools that are
+installed in the LibreCores CI environment. Beside the tool
+information we provide the files to reproduce the build or
+installation of tools using [docker](http://docker.io).
+
+**Why set up your own tools environment?**
+
+Setting up your own tools environment using our docker images can be
+useful as a fast way of getting your environment set up. Beside that
+it can be useful to reproduce problems with your build in LibreCores
+CI.
+
+## How to
+
+In the following we will briefly introduce to you how to setup your
+tools environment on a local path called `/tools`. The idea is
+basically that you run the docker images for the tools that you need
+to be installed. For example you would install FuseSoC using:
+
+    docker run librecores/tools-fusesoc -v /tools:/tools
+
+This will run the GCC build for the `latest` tag and map the host
+machine folder `/tools` to the docker folder `/tools`.
+
+You can select another tag, e.g.,
+
+    docker run librecores/fusesoc:1.6 -v /tools:/tools
+
+Finally, each tool can have extra parameters that you find with the
+tool information below. In the case of the RISC-V cross compiler for
+example the build jobs can be set:
+
+    docker run librecores/tools-riscv-gcc:20170209 -v /tools:/tools -e JOBS=-j16
+
+## Environment Modules
```

Comment:
It could be a collision of the generic docker tools & LibreCores requirements.
Maybe we need to generalize it